defaults.ts

```diff
@@ -0,0 +1,3 @@
+export const defaults = {
+  avalancheCenter: 'NWAC',
```

Comment:
WDYT about putting this in app.json: https://github.com/stevekuznetsov/avalanche-forecast/blob/8bbd2804510d9e5d50fc151dac576443c9735080/app.json#L51-L56 (don't feel strongly, easy enough to move it later)

Reply:
Done